services/src/handlers/plots.rs

```diff
@@ -294,7 +294,7 @@ mod tests {
         ];
         let req = actix_web::test::TestRequest::get()
             .uri(&format!(
-                "/plot/{}/?{}",
+                "/plot/{}?{}",
```

Comment:
why does our API change with this PR?

Reply:
it doesn't change, the test does. In production we already use it without a terminating slash.